requirements.txt

```diff
@@ -18,3 +18,6 @@ click
 
 # Excel File Manipulation
 openpyxl
+
+# for EpubLoader
+epub2txt
```

Comment:
Why not use UnstructuredEPubLoader from langchain?

Reply:
Right good suggestion. But somehow UnstructuredEPubLoader('....epub') always hangs in my setup. 

I have  Windows 10, python 3.10, the latest langchain. 

I tried all sorts of epub files，a non-existent epub file (straight from the langchain doc `loader = UnstructuredEPubLoader("winter-sports.epub"`). It always hangs indefinitely. Even Ctrl-C doesn't work.

My EpubLoader will always happily load or raise an exception for a non-existent file.

Reply:
@ffreemt In [my project](https://github.com/sime2408/scrapalot-chat/blob/main/scripts/app_utils.py#L52), it is working just fine, but not in the way that I provide name of the file through constructor 👀

Reply:
OK I tested UnstructuredEPubLoader in Linux. It runs OK.

However, UnstructuredEPubLoader parses an epub file to only one single doc hence the metadata is just 
```
[{'source': 'docs\\test.epub'}]
``` 
while EpubLoader'metadata parses an epub file to several docs with chapter info when available. The metadata is something like:
```
[{'source': 'docs\\test.epub', 'ch.': 'NA'},
 {'source': 'docs\\test.epub', 'ch.': 'NA'},
 {'source': 'docs\\test.epub', 'ch.': '1'},
 {'source': 'docs\\test.epub', 'ch.': '1*'},
 {'source': 'docs\\test.epub', 'ch.': '2'}, ...
```